PokeDemo/Models/Pokemon.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  Pokemon.swift
+//  PokeDemo
+//
+//  Created by Rudr Bansal on 05/07/22.
+//
+
+import Foundation
+
+struct Pokemon: Decodable, Equatable {
+    var name: String?
+    var url: String?
```

Comment:
why optionals?

Reply:
These are coming in response, so kept them optional.

Reply:
Remove optionals. If any response does not come with value handle as JSON decode error

Reply:
Covered in 8e62e9f9cf6903a304d206bb02281454063764a0